src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -33,6 +35,7 @@ export default class GraphFilter extends React.Component<GraphFilterProps, Graph
   updateDuration = (value: number) => {
     if (this.props.graphDuration.value !== value) {
       // notify callback
+      sessionStorage.setItem('appDuration', JSON.stringify(value));
```

Comment:
I'm not familiar with sessionStorage, so perhaps some comments from my side are pretty obvious.
Why to stringify the value ? Is it a necessary serialization ?

Reply:
You can save a number in the sessionStorage, must be a string (https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem) so I set stringify to convert and avoid future errors (https://www.w3schools.com/js/js_json_stringify.asp) we could cast the number to string only too but I think that this will be a json object in a future.